apps/teams-test-app/src/components/utils/ApiContainer.tsx

```diff
@@ -9,7 +9,7 @@ export interface ApiContainerProps {
 export const ApiContainer = (props: React.PropsWithChildren<ApiContainerProps>): React.ReactElement => {
   const { children, name, result } = props;
 
-  if (!name || !/^[a-zA-Z0-9]+$/.test(name)) {
+  if (!name || !/^[a-zA-Z0-9.]+$/.test(name)) {
     throw new Error('name has to be set and it can only contain alphanumeric characters.');
```

Comment:
Maybe we can add that you can have dots in the name?

Reply:
I think that's more of an exception and the only reason I added it here is not to have to change all the existing tests. So maybe it's ok to keep the message a bit more restrictive than the actual check to discourage folks from adding more names with dots?